website/content/docs/agent-and-proxy/proxy/caching/index.mdx

```diff
@@ -36,6 +37,21 @@ specific scenarios.
    that are issued using the tokens managed by the proxy, will be cached and
    its renewals are taken care of.
 
+## Static secret caching
+
+Vault Proxy can be configured to cache KVv1 and KVv2 secrets as well as dynamic secrets described
+above. This means that, when enabled, subsequent requests to the same KV secret made via Proxy will
+only result in a single request made to Vault. Proxy will keep a single cache entry of the secret in its
+cache, and will only provide the cached response to requests made with tokens that can access the secret.
```

Comment:
```suggestion
You can configure Vault Proxy to cache dynamic secrets and static, KVv1 and KVv2
secrets. When you enable caching for static secrets. Proxy keeps a cached entry
of the secret but only provides the cached response to requests made with tokens
that can access the secret. As a result, multiple requests to Vault Proxy for
the same KV secret only require a single, initial request made to Vault.
```

Style correction: write in active voice, speak directly to the reader, avoid "this" as a pronoun

Reply:
Thanks! I slightly reworded the last sentence to hopefully be clearer:
initial request made to Vault
->
initial request to be forwarded to Vault

To hopefully make it clear that Proxy does the forwarding for you, and the client doesn't need to send the request to Vault itself

Reply:
I'll leave this un-resolved to give you a chance to review the updated wording -- let me know if you disagree and I can change it back :)